app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   def create
     return render status: 403, text: 'Token missing' if params[:token].blank?
     token = Doorkeeper::AccessToken.by_token(params[:token])
-    is_admin = token.resource_owner.roles.where(name: 'admin').count > 0
+    is_admin = token.resource_owner.roles.where(name: 'admin').positive?
```

Comment:
It didn't tell you to change `.count > 0`, it told you to change `> 0`

Anyways, the better option is `token.resource_owner.roles.where(name: 'admin').exists?`

Reply:
oh jeez good catch, yeah i should have just removed the `> 0` but will change to what you wrote above.